src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -57,7 +57,8 @@
     "conflict": {
         "phpdocumentor/reflection-docblock": "<3.0",
         "phpdocumentor/type-resolver": "<0.2.0",
-        "symfony/console": "<3.3"
+        "symfony/console": "<3.3",
+        "symfony/web-profiler-bundle": "<3.3"
```

Comment:
That's what I figured in [your comment ](https://github.com/symfony/symfony/pull/21375#issuecomment-274495087).
The fix is coming.